views/components/App.jsx

```diff
@@ -49,52 +55,96 @@ export default class App extends React.Component {
     });
   }
 
+  selectInput (e) {
+    var input = WebMidi.getInputById(e.target.value);
+    input.addListener('controlchange', "all", this.midiRxCC)
+    this.setState({
+      currentInput: input,
+    });
+  }
+
+  midiRxCC (e) {
+    console.log("RXed " + e);
+    // change Rx to true to cause LED to blink
+    this.setState({inputRx: true});
+  }
```

Comment:
so right now the light turns on as soon as midi starts, and never turns off.  We probably want to turn the light off after some period of not seeing any midi signals.  So, i'd put a fn that will turn it off after a timeout:

```
midiRxCC () {
  this.setState({inputRx: true});
  // remove old listener (no-op if this.rxListener is undefined)
  clearTimeout(this.rxListener);
  // set a listener to turn off the light.
  this.rxListener = setTimeout(() => this.setState({inputRx: false}), 100);
}
```